### ISSUE TITLE
Eliminate "name" column from dataset table

### ISSUE THREAD

**Issue by JoanneBogart** [LSSTDESC/dataregistry#9]
The name column serves no useful function.  Better to encourage people to use aliases if they need a nickname

**Comment by stuartmcalpine** [comment]
Should version also be added to the relative path? If a user makes a new dataset with the same name, but an updated version, they would need to chose a new relative path?

**Comment by JoanneBogart** [comment]
I don't think so.  In fact I can imagine cases in development where someone wants to overwrite the old one. If the dataset is not overwritable and they give the same path the code currently prints out an error and returns without doing anything.  (Should be changed to raise an exception.)

**Comment by JoanneBogart** [comment]
With the merge of the branch u/jrbogart/dependencies this issue has been addressed.
